manipulation/packages/arm_pkg/CMakeLists.txt

```diff
@@ -16,9 +16,6 @@ find_package(ament_cmake_python REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclpy REQUIRED)
 find_package(pcl_conversions REQUIRED)
-find_package(pcl_msgs REQUIRED)
-find_package(OpenCV REQUIRED)
-find_package(Eigen3 REQUIRED)
```

Comment:
you might need to remove [this](https://github.com/RoBorregos/home2/pull/214/files#:~:text=%40%40%20%2D80%2C24%20%2B75%2C24%20%40%40-,target_link_libraries(gpd_test,),-%23%20Install%20targets) if you remove this